### PR TITLE
Generalize proposal

### PR DESCRIPTION
**Pull request by marcoscaceres**
People reading the proposal are fixating on the "modal" part, which is not important. The thing that matters is that we need some kind of general handler registration and presentation model.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,4 @@
-# Modal Window
+# Handlers
+This is the repository captures general requirements for "handlers" for the Web. These are web applications that are able to handle one or more actions (intents?) in coordination with the end user.
```
Comment (matdehaast) [inline README.md]:
```suggestion
This repository captures general requirements for "handlers" for the Web. These are web applications that are able to handle one or more actions (intents?) in coordination with the end user. 
```

**Comment by marcoscaceres** [comment]
Thanks @matdehaast for catching that :)

**Comment by rsolomakhin** [comment]
Does the word "handler" imply a service worker here? I've received feedback that service worker installation (for Payment Handler, for example) creates overhead, so it would be simpler if `https://alice.com/index.html` could write `window.openModal('https://bob.com')` directly.

**Comment by marcoscaceres** [comment]
> Does the word "handler" imply a service worker here?

no - certainly not. But it doesn't preclude it either :)  

> I've received feedback that service worker installation (for Payment Handler, for example) creates overhead, so it would be simpler if https://alice.com/index.html could write window.openModal('https://bob.com') directly.

Agree.

**Review by adrianhopebailie — APPROVED** [review]

**Pull request merged**